templates/docs/examples/patterns/search-and-filter/with-chips.html

```diff
@@ -0,0 +1,121 @@
+
+{% block content %}
+<div class="p-search-and-filter">
+  <div class="p-search-and-filter__search-container" aria-expanded="false" data-active="true" data-empty="false">
+    <div class="p-chip" aria-pressed="true" role="button">
+      <span class="p-chip__lead">CLOUD</span><span class="p-chip__value">Google</span>
+      <button class="p-chip__dismiss"><i class="p-icon--close">Dismiss</i></button>
+    </div>
+    <div class="p-chip" aria-pressed="true" role="button">
+      <span class="p-chip__lead">CLOUD</span><span class="p-chip__value">AWS</span>
+      <button class="p-chip__dismiss"><i class="p-icon--close">Dismiss</i></button>
+    </div>
+    <form class="p-search-box" data-overflowing="false">
+      <label class="u-off-screen" for="search">Search and filter</label>
+      <input autocomplete="off" class="p-search-box__input" id="search" name="search" placeholder="Search and filter" type="search" value="">
+      <button alt="search" class="u-off-screen" type="submit">Search</button>
+    </form>
+  </div>
+  <div class="p-search-and-filter__panel" aria-hidden="true">
+    <div class="p-filter-panel-section">
+      <h3 class="p-filter-panel-section__heading">Cloud</h3>
+      <div class="p-filter-panel-section__chips" aria-expanded="false">
+        <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+          <span class="p-chip__lead">CLOUD</span><span class="p-chip__value">Google</span>
+        </div>
+        <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+          <span class="p-chip__lead">CLOUD</span><span class="p-chip__value">AWS</span>
+        </div>
+        <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+          <span class="p-chip__lead">CLOUD</span><span class="p-chip__value">Azure</span>
+        </div>
+      </div>
+    </div>
+
+    <div class="p-filter-panel-section">
+      <h3 class="p-filter-panel-section__heading">Region</h3>
+      <div class="p-filter-panel-section__chips" aria-expanded="false">
+        <span>
+          <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+            <span class="p-chip__lead">REGION</span><span class="p-chip__value">us-east1</span>
+          </div>
+        </span>
+        <span>
+          <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+            <span class="p-chip__lead">REGION</span><span class="p-chip__value">us-north2</span>
+          </div>
+        </span>
+        <span>
+          <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+            <span class="p-chip__lead">REGION</span><span class="p-chip__value">us-south3</span>
+          </div>
+        </span>
+        <span>
+          <div class="p-chip" aria-pressed="false" role="button" tabindex="0">
+            <span class="p-chip__lead">REGION</span><span class="p-chip__value">us-north4</span>
+          </div>
+        </span>
+        <span>
```

Comment:
Are these spans needed? Is this the markup from react component?
I think having div (block element) inside span (inline) is not valid HTML.

Reply:
That would be react, they are gone